src/sequence_of_play.tex

```diff
@@ -281,9 +202,190 @@ \subsubsection*{\zsavepos{arrowPeaceSeqStart}D. Peace Resolutions}
+\end{itemize}
+\smallheading{Optional Rule 7: Revealing More (p.~39)}
+\begin{itemize}
+	\item If 6 PRs, reveal 4 of them
+\end{itemize}
+\subsubsection*{B. Draw Action Cards}
+
+\begin{itemize}
+	\item Draw 3 cards (may draw one by one)
+\end{itemize}
+
+\subsubsection*{C. Pay for Action Cards}
+\begin{itemize}
+	\item 2 \ducats per drawn card you keep
+\end{itemize}
+
+\subsubsection*{D. Pick/Replace Missions}
+\begin{itemize}
+	\item Mission can be picked if 1+ prerequisite Missions are completed (p.~42)
+	\item Max 2 in hand
+\end{itemize}
+
+\subsection*{2. Actions}
+\begin{itemize}
+	\item On each Turn, must take 1 Action, \conj{and}
+	\item May take any \# of Minor Actions
+	\botrule{\item Bots use Main Turn Structure (p.~7)}
+\end{itemize}
+\subsubsection*{Passing}
+\begin{itemize}
+	\item Allowed only if \action{Event} taken
+	\item May take Minor Actions on same Turn
+	\item First 2/3/3/4 PRs to Pass gain \ducats
+	\item If no more PRs eligible to gain \ducats,\\ \strong{End of Action Phase} is triggered
+	\begin{itemize}
+		\item Each remaining PR may take 1 more Action of their choice, \conj{then}
+		\item PRs who still have not taken \action{Event},\\ get one more Turn to take \action{Event}
+		\item If PR has taken their final Turn, they are considered to have Passed
+	\end{itemize}
+\end{itemize}
+
+\subsection*{3. Peace \& Rebels}
+\subsubsection*{A. Remove Casus Belli \& Truces}
+\begin{itemize}
+	\item Lose \p2 if not at war with the CB target
+\end{itemize}
+
+\subsubsection*{B. NPR Invasions \normal{(advanced rules)}}
+\begin{itemize}
+	\item \hspacezero\zsavepos{arrowNprInvasionsStart}NPRs at War might invade PRs
+	\botrules
+	\item Bots suffer max 1 NPR invasion. Priority: 1. most \rebels; 2. \az (p.~2)
+\end{itemize}
+
+\subsubsection*{C. Rebels Siege \& Move}
+\begin{itemize}
+	\item Resolve by Area: 1. most \rebels; 2. \az
+	\item In \strong{Areas with \unrest}, Rebels Siege \unrest. Priority:
+	\begin{enumerate}
+		\item Large Provinces
+		\item If 2+ PRs with \unrest in the Area, PR with lowest Tax Income (no tie) in the Area chooses (p.~37)
+		\item \az
+	\end{enumerate}
+	\item In \strong{Areas with no \unrest}
+	\begin{itemize}
+		\item If an Area with \unrest is Adj. by Land \conj{and} \rebels will not be outnum. there by PRs
+		\begin{itemize}
+			\item Move all but 1 \rebel. Priority:\\ 1. most \unrest; 2. most PR \towns; 3. \az
+			\item Resolve Battle immediately
+		\end{itemize}
+		\item Otherwise remove 1\rebel
+	\end{itemize}
+\end{itemize}
+
+\subsubsection*{\zsavepos{arrowPeaceSeqStart}D. Peace Resolutions\zsavepos{arrowPeaceTermsStart}}
+\begin{itemize}
+	\item When Emperor makes peace with an External Aggressor (p.~44-45)
+	\begin{itemize}
+		\item If Emp. won \conj{and} no HRE Prov. is lost
+		\begin{itemize}
+			\item Raise \authority by \# of Areas joining HRE
+		\end{itemize}
+		\item If Emp. lost
+		\begin{itemize}
+			\item Drop \authority by \# of Areas leaving HRE
+		\end{itemize}
+	\end{itemize}
```

Comment:
I modified the Truce sentence to make it fit, let me know it this is understandable.
Otherwise, an alternative solution would be to squeeze the Peace Sequence frame to 29.2 base line skips, which is not so nice but might be acceptable.

Reply:
Not sure if "resp." would be understandable. Tried a few more alternative wordings and all the following ones fit on 1 line:

- Flip War token(s) when Peace is made
- Making Peace flips War token(s) to Truce
- Making Peace flips respective War token(s)

Do you like any of these?

Reply:
Also noticed now that "Peace" is not capitalized in "When Emperor makes peace...". This is a game term that is always capitalized in the rulebook. Let's do the same here

Reply:
Great suggestions, done